qiskit/test/mock/fake_backend.py

```diff
@@ -38,6 +62,257 @@ def __init__(self, token="123456", url="https://"):
         self.project = "project"
 
 
+class FakeBackendV2(BackendV2):
+    """This is a dummy bakend just for resting purposes. the FakeBackendV2 builds on top of the BackendV2 base class."""
+
+    dirname = None
+    conf_filename = None
+    props_filename = None
+    defs_filename = None
+    backend_name = None
+
+    def __init__(self):
+        self._conf_dict = self._get_conf_dict_from_json()
+        self._props_dict = self._set_props_dict_from_json()
+        self._defs_dict = self._set_defs_dict_from_json()
+        super().__init__(
+            provider=None,
+            name=self._conf_dict.get("backend_name"),
+            description=self._conf_dict.get("description"),
+            online_date=self._conf_dict.get("online_date"),
+            backend_version=self._conf_dict.get("backend_version"),
+        )
+        self._target = None
```

Comment:
Is there a reason to do this lazily? It should be fairly quick to deserialize the json and create target.

Reply:
This was based on the provider team PR: https://github.com/Qiskit/qiskit-ibm-runtime/pull/102 which you have pointed out in our meetings that it has different considerations with the mocked backends. Now I understand that that PR uses lazy loading because fetching configuration, properties, pulse defaults from devices may take sometime. I will modify that.